browser/resources/settings/brave_content_page/containers.html.ts

```diff
@@ -71,18 +84,59 @@ export function getHtml(this: SettingsBraveContentContainersElement) {
           >
             <div slot="title">${this.getEditDialogTitle_()}</div>
             <div slot="body">
-              <div class="flex">
-                <cr-input
-                  .value="${this.editingContainer_.name}"
-                  @input="${this.onContainerNameInput_}"
-                  autofocus
-                  required
-                  auto-validate
-                  pattern="^.*\\S.*$"
-                  label="$i18n{SETTINGS_CONTAINERS_CONTAINER_NAME_LABEL}"
-                  placeholder="$i18n{SETTINGS_CONTAINERS_CONTAINER_NAME_PLACEHOLDER}"
-                >
-                </cr-input>
+              <div class="flex editing-container">
+                <section class="name-section">
+                  <div class="edit-subsection-label">
+                    $i18n{SETTINGS_CONTAINERS_CONTAINER_NAME_LABEL}
+                  </div>
+                  <cr-input
+                    .value="${this.editingContainer_.name}"
+                    @input="${this.onContainerNameInput_}"
+                    autofocus
+                    required
+                    auto-validate
+                    pattern="^.*\\S.*$"
+                    placeholder="$i18n{SETTINGS_CONTAINERS_CONTAINER_NAME_PLACEHOLDER}"
+                  ></cr-input>
+                </section>
+                <section class="background-colors-section">
+                  <div class="edit-subsection-label">
+                      $i18n{SETTINGS_CONTAINERS_CONTAINER_COLOR_LABEL}
+                  </div>
+                  <div class="background-colors-list">
+                    ${backgroundColors.map(
+                      (color) => html`
+                        <settings-brave-content-containers-background-chip
+                          ?selected="${color.value
+                          === this.editingContainer_!.backgroundColor.value}"
+                          background-color="${skColorToHexColor(color)}"
+                          @background-selected="${this
+                            .onContainersBackgroundColorSelected_}"
+                        ></settings-brave-content-containers-background-chip>
+                      `,
+                    )}
+                  </div>
+                </section>
+                <section class="icons-section">
+                  <div class="edit-subsection-label">
+                      $i18n{SETTINGS_CONTAINERS_CONTAINER_ICON_LABEL}
+                  </div>
+                  <div class="icons-list">
+                    ${[...iconLeoMap.entries()].map(
+                      (icon: [Icon, string]) => html`
```

Comment:
nit - might be worth destructing this if we keep it
```suggestion
                      ([icon, leoIcon]: [Icon, string]) => html`
```